tests/test_utils.py

```diff
@@ -28,3 +45,134 @@ def raises_type_error():
     with raises(TypeError) as excinfo:
         raises_type_error()
     assert excinfo.value == mock_type_error
+
+
+def test_order_transactions(bdb_driver, alice_keypair, bob_keypair):
+    import random
+    from coalaip_bigchaindb.utils import make_transfer_tx, order_transactions
+    create_tx = bdb_driver.transactions.prepare(
+        operation='CREATE',
+        signers=alice_keypair['public_key'])
+    transfer_to_bob_tx = make_transfer_tx(bdb_driver, input_tx=create_tx,
+                                          recipients=bob_keypair['public_key'])
+    transfer_back_to_alice_tx = make_transfer_tx(
+        bdb_driver, input_tx=transfer_to_bob_tx, recipients=alice_keypair['public_key'])
+
+    correct_order = [create_tx, transfer_to_bob_tx, transfer_back_to_alice_tx]
+    for _ in range(0, 100):
```

Comment:
😏  *just to be safe...*

On average it should take 36 runs to test all 6 permutations. I guess I could use an array permutation lib that's probably within easy reach, but these tests run so fast anyway.